compiler/rustc_serialize/src/opaque.rs

```diff
@@ -186,12 +186,12 @@ impl FileEncoder {
     pub fn with_capacity<P: AsRef<Path>>(path: P, capacity: usize) -> io::Result<Self> {
         // Require capacity at least as large as the largest LEB128 encoding
         // here, so that we don't have to check or handle this on every write.
-        assert!(capacity >= max_leb128_len());
+        assert!(capacity >= max_max_leb128_len());
```

Comment:
maybe just inline the function instead of having a `max_max_` function?

Reply:
I dunno, inlining seems like magic ~~constants~~ types

Reply:
true... but then maybe name it `largest_max_leb128_len()`?

Reply:
sure, done